### PR TITLE
Math functions

### PR DESCRIPTION
**Pull request by zrzka**
**Breaking changes**

* `NOW()` function accepts one `boolean` argument only
* Python style kwargs were removed in favor of positional arguments
  * Functions & filters no longer accepts them, only positional arguments

**Additions**

* `POW(base, exponent)` function introduced
* `LOG10(x)` function introduced

### CODE REVIEW

**Comment by zrzka** [comment]
@resin-ci retest

**Comment by zrzka** [comment]
@resin-ci retest

File: src/builtin/function/math.rs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    validate_args_len(args, 1, "LOG10")?;
+
+    let x = get_one_arg(args, Value::as_f64)?;
+    Ok(Value::Number(Number::from_f64(x.log10()).ok_or_else(|| {
+        Error::with_message("expressions results to NaN").context("function", "LOG10")
+    })?))
+}
+
+#[cfg(test)]
+mod tests {
```
Comment (cyplo) [inline src/builtin/function/math.rs]: These are good example-based tests, I think in the future they could really benefit from being generalized to property-based, using `proptest` for example.

**Comment by cyplo** [comment]
@resin-ci retest

**Pull request merged**